ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -188,7 +188,7 @@ void addReplica(PartitionId partitionId) throws ReplicationException {
             tokenHelper.getFindTokenFactoryFromReplicaType(peerReplica.getReplicaType());
         RemoteReplicaInfo remoteReplicaInfo =
             new RemoteReplicaInfo(peerReplica, cloudReplica, store, findTokenFactory.getNewFindToken(),
-                storeConfig.storeDataFlushIntervalSeconds * SystemTime.MsPerSec * Replication_Delay_Multiplier,
+                storeConfig.storeDataFlushIntervalSeconds * TimeUnit.SECONDS.toMicros(1) * Replication_Delay_Multiplier,
```

Comment:
Generally it's preferred to use this pattern: `TimeUnit.SECONDS.toMillis(frontendConfig.cacheValiditySeconds)` instead of multiplying times one unit.

This will handle some overflow cases and makes the code more concise.

Also, I noticed that microsecond (10E-6) conversions are used where milliseconds (10E-3) (`TimeUnit::toMillis`) is needed. Please double check the places where `toMicros` is used.

Reply:
@cgtz thanks for review.
> Generally it's preferred to use this pattern: TimeUnit.SECONDS.toMillis(frontendConfig.cacheValiditySeconds) instead of multiplying times one unit.
- Previously `SystemTime.MsPerSec` was equivalent to `TimeUnit.SECONDS.toMicros(1)`, so I've changed in this way. Is `frontendConfig.cacheValiditySeconds` a new feature?

> I noticed that microsecond (10E-6) conversions are used where milliseconds (10E-3) (TimeUnit::toMillis) is needed. Please double check the places where toMicros is used.
- Parameter for `storeConfig.storeDataFlushIntervalSeconds * TimeUnit.SECONDS.toMicros(1)` is used as `tokenPersistIntervalInMs` in `RemoteReplicaInfo`, and seems it should be `milli-second` as you commented. But just bit worried that this change can effect other logics. Is it okay to change on?(because I couldn't find document/comment about 'RemoteReplicaInfo' so not sure of this...)

Reply:
@cgtz thanks for the feedback. Updated~